cmake/Modules/GetGitRevisionDescription.cmake.in

```diff
@@ -1,11 +1,15 @@
-# Copyright (C) 2009 - 2010 Ryan Pavlik <rpavlik@iastate.edu> <abiryan@ryand.net>
```

Comment:
we could add these, but we still should keep the rest intact

Reply:
Now its gone and doesn't matter where it is, just that it's there